auraed/src/cells/cell_service/cell_service.rs

```diff
@@ -237,6 +241,29 @@ impl CellService {
         executables.broadcast_stop().await;
         Ok(())
     }
+
+    #[tracing::instrument(skip(self))]
+    async fn list(&self) -> Result<CellServiceListResponse> {
+        let cells = self.cells.lock().await;
+
+        
+        Ok(CellServiceListResponse { 
+            cells: cells.entries()
+                        .iter()
+                        .map(|cell| 
+                            Cell { 
```

Comment:
there might be a very good reason for this: why do we need this map at all? or why is in not

```
.map(|cell| cell.clone())

Reply:
That's a name collision. One is `auraed::cells::cell_service::cells::cell` and the other is `Cell` as defined in the proto.